deploy/playbooks/inventories/group_vars/aws.yml

```diff
@@ -0,0 +1,12 @@
+---
+#
+# SPDX-License-Identifier: MIT
+#
+# Copyright (c) 2020 The Authors.
+# Authors: Bin Liang  <@liangbin>
+#
+# Summary: Group specific playbook variables for Inventory group named "aws"
+#
+ansible_port: 22
+ansible_user: "ubuntu"
+ansible_ssh_private_key_file: "~/.ssh/Bin-ssh-keypair-testVM.pem"
```

Comment:
Can we set this key in the env-setup.sh script?

Reply:
They are now